tests/10-api.spec.js

```diff
@@ -1,18 +1,88 @@
+/* eslint-disable max-len */
 /*!
  * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
  */
-import * as middleware from '..';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import express from 'express';
 import {authorizeZcapInvocation} from '..';
+import didIo from 'did-io';
+import didKeyDriver from 'did-method-key';
+import jldl from 'jsonld-document-loader';
+
+const _documentLoader = new jldl.JsonLdDocumentLoader();
+
+didIo.use('key', didKeyDriver.driver());
+
+async function documentLoader(url) {
```

Comment:
I think using https://github.com/digitalbazaar/security-document-loader#usage here would be easier. (It already comes pre-loaded with did-key driver, and the docs show how to add other contexts (like the zcap one)

Reply:
Used @digitalbazaar/security-document-loader 3ca48bb